Riku/src/Frontend/GUICallbacks.h

```diff
@@ -47,27 +38,32 @@ namespace CEGUI::Functor {
     {
     private:
         std::string guiName;
+        CEGUI::GUI*& activeGUI;
```

Comment:
Przyznam się, że to wygląda trochę niecodziennie. Czy zwykły wskaźnik by nie wystarczył?

Reply:
Nie może to być zwykły wskaźnik ze względu na linijkę 50
```cpp
activeGUI = guiDic[guiName];
```
Gdyby to był normalny wskaźnik w tym miejscu zastąpilibyśmy go innym lokalnie, ale oryginalny activeGui w Scene pozostałby niezmieniony. Z tego względu potrzebujemy albo referencji do wskaźnika, albo podwójnego wskaźnika. Według mnie referencja do wskaźnika jest ładniejsza

Reply:
Rozumiem, to ok.

Reply:
W dwóch miejscach referencja faktycznie nie była potrzebna więc ją usunąłem. W pozostałych jest ona konieczna